cms/djangoapps/modulestore_migrator/tasks.py

```diff
@@ -833,14 +623,14 @@ def bulk_migrate_from_modulestore(
                         f"{MigrationStep.IMPORTING_STRUCTURE.value}"
                     )
                     change_log, root_migrated_node = _import_structure(
-                        source_data.migration,
-                        source_data,
-                        target_library,
-                        content_by_filename,
-                        root_node,
-                        status,
+                        source_data=source_data,
+                        target_library=target_library,
+                        content_by_filename=content_by_filename,
+                        root_node=root_node,
+                        status=status,
                     )
                     source_data.migration.change_log = change_log
+                    source_data.migration.save()  # @@TODO keep or nah?
```

Comment:
My understanding is that saving here ensures that we persist the association between the migration and the changelog. 
```suggestion
                    source_data.migration.save()
```